content/techniques/sql.md

```diff
@@ -743,6 +743,10 @@ TypeOrmModule.forRootAsync({
 
 This construction works the same as `useClass` with one critical difference - `TypeOrmModule` will lookup imported modules to reuse an existing `ConfigService` instead of instantiating a new one.
 
+If you don't use ConnectionName as 'default', you need to specify ConnectionName so that nestjs can recognize it.
+> **Hint**: make sure the `name` property is on the same level as the `useFactory`, `useClass`, or `useValue` property. This will allow Nest to properly create the injection token
```

Comment:
```suggestion

> info **Hint**: make sure the `name` property is on the same level as the `useFactory`, `useClass`, or `useValue` property. This will allow Nest to properly create the injection token
```

Sorry, had the wrong syntax. This is the last edit, I promise. This will make the hint show up in blue like it does on Line #645